### ISSUE TITLE
Does not receive event for oauth url redirect

### ISSUE THREAD

**Issue by lukepighetti** [avioli/uni_links#40]
Hi there,

if you install `url_launcher` and `uni_links` and open up a stream to `uni.getUriLinksStream().listen(print);` and then `url.launch("https://myserver/myredirect")` which redirects to a scheme, such as `myscheme:/?payload="foo"`, you will see the browser disappear (as expected) but no event is recoreded by `uni_links`. However, if you open your browser manually, and type  `myscheme:/?payload="foo"` in the address bar, it will open the app and `uni_links` will fire the event to console.

Short version: if you're using `uni_links` to listen for oauth2 redirects, the events do not fire. But if you use it any other way, it seems to work.

Do you have any suggestions for listening for these events?

**Comment by mohamedazher** [comment]
Hi,

I am having the exact same issue while trying to implement oauth. Were you able to solve this issue?
